department-of-veterans-affairs/vets-website
# Revert to old address ui schema

## Summary
Revert dependentChildAddress page address ui schema to old implementation. (Bug around State input discovered with address web component pattern and arrays)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Sign in via mocked authentication at `http://localhost:3001/sign-in/mocked-auth`
3. Copy maximal-data.json into 'Save in progress' menu and return to /dependents page
4. Continue to the 'Does your child live with you?' page and select 'No'
5. Verify that the state field is a select input

## Related issue(s)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots
![localhost_3001_pension_application_527EZ_household_dependents_children_address_0](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/b1e43fd2-b084-4599-8cdd-09b55e620d75)


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
n/a

### Quality Assurance & Testing

### Error Handling


### Authentication
